src/button/button-icon.scss

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+@mixin iui-button-icon {
+  &:not(.iui-user-icon) {
```

Comment:
no need for `:not(iui-user-icon)` anymore. This was only added because the specificity of `iui-button > iui-icon` was too high.